biojava-protein-comparison-tool/pom.xml

```diff
@@ -58,7 +58,7 @@
 		<dependency>
 			<groupId>net.sourceforge.jmol</groupId>
 			<artifactId>jmol</artifactId>
-			<version>13.0.14</version>
+			<version>14.31.10</version>
```

Comment:
This is a major version upgrade, and a large jump at that.  Are there any build or runtime issues caused by this?

Reply:
@heuermh Unfortunately we don't have any tests for jmol functionality. I'm not even sure how that would work, other than manually running the alignment tools and/or ce-symm. However I think it's still a good idea to keep jmol recent. (I build the jmol mvn packages for at least the final release of each major version).